src/storage/storageController.ts

```diff
@@ -58,11 +58,11 @@ export default class StorageController {
     storageScope?: StorageScope
   ): Thenable<void> {
     if (storageScope === StorageScope.WORKSPACE) {
-      this._workspaceState.update(variableName, value);
+      void this._workspaceState.update(variableName, value);
```

Comment:
can't this thing fail? what is the issue with waiting?

Reply:
@Anemy probably you can answer this one?

Reply:
I remember it being flaky on ci, sometimes it would not resolve or error and would just stall. The value would still update though.